core/src/ballot_box.rs

```diff
@@ -13,28 +13,25 @@ use crate::{constants::PRECISE_CONSENSUS, discriminators::Discriminators, error:
 #[repr(C)]
 pub struct Ballot {
     merkle_root: [u8; 32],
-    max_total_claim: PodU64,
-    max_node_count: PodU64,
+    is_cast: PodBool,
```

Comment:
would this make more sense on the OperatorVote? now that ballots are one to many

Reply:
Took it out